javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,193 @@
+(function () {
+  "use strict";
+  // ###### 1.1. Doubling of number
+
+  let numbers = [1, 2, 3, 4];
+  let newNumbers = [];
+
+  numbers.forEach((i) => {
+    if (i % 2 !== 0) {
+      newNumbers.push(i * 2);
+    }
+  });
+
+  // console.log("The doubled numbers are", newNumbers); // [2, 6]
+
+  // ###### 1.2. Working with movies
+
+  // STEP: 1 & 2
+  document
```

Comment:
Code splitting: I suggest you create helpers outside of the scope of the DOM manipulation functions.
There are two options: 

-  In the same file, write the functions outside of the scope of the event listeners, and call those functions from inside of them. 

```
function creatingTitleLength( ) { ... }
 
mySelector.addEventListener("click", ( ) {
      ...
      creatingTitleLength( );
      ...
}
```

- You can create new files, write your functions in them, export them and use them wherever they are needed.

Reply:
actualy i tried it but it gave always error so i did in a way it works :D